examples/jf-control-app/commands/example/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -457,6 +457,70 @@ ExampleOperationalCredentialsIssuer::GenerateNOCChain(const ByteSpan & csrElemen
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ExampleOperationalCredentialsIssuer::SignICAC(const ByteSpan & icaCsr, MutableByteSpan & icac)
```

Comment:
See my comment https://github.com/project-chip/connectedhomeip/pull/39362#discussion_r2166069655 - I think we need an additional input parameter for this function, uint64_t anchorFabricId.